R-Interface/jasprcpp.cpp

```diff
@@ -98,7 +101,10 @@ void STDCALL jaspRCPP_init(const char* buildYear, const char* version, RBridgeCa
 	_stringNativeToUtf8		= nativeToUtf8;
 
 	jaspRCPP_logString("Creating RInside.\n");
+
 	rinside = new RInside();
+
+	R_TempDir = (char*)tempDir;
```

Comment:
Don't understand: this variable does not seem to be used.

Reply:
If you check where it is defined you'll see it is `extern` and it is in fact in R.
There were some issues on windows where the environment variables were converted to local codepage... Which messed up stuff if it couldnt fit. So I now just worked around it like this.

But actually, I think setting the active codepage to utf-8 probably already solved that. 

However, it works now and we can always clean stuff up later.

Reply:
Previously this value was shared through `TMPDIR`